R/plot_carp.R

```diff
@@ -76,21 +77,19 @@
 #' @examples
 #' \dontrun{
 #' carp_fit <- CARP(presidential_speech)
-#' plot(carp_fit, type='interactive')
+#' plot(carp_fit, type='heatmap', interactive=TRUE)
```

Comment:
Could you add some more examples to this documentation? I'd keep the heatmap inside a `dontrun` block since it's slow, but the static `ggplot` stuff can be a "real" example.